swap_meet/vendor.py

```diff
@@ -1,2 +1,50 @@
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory == None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category):
+        self.category = category
+        category_list = []
+        for item in self.inventory:
+            if item.category == category:
+                category_list.append(item)
+        
+        return category_list
+
+    def swap_items(self, vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in vendor.inventory:
+            return False
+
+        self.inventory.remove(my_item)
+        vendor.inventory.append(my_item)
```

Comment:
These would've been good places to use your own `add` and `remove` functions that you created earlier.